src/BTrees/_base.py

```diff
@@ -582,7 +614,68 @@ def merge_output(it):
     def __repr__(self):
         return self._repr_helper(self.items())
 
-class Set(_BucketBase):
+class _MutableSetMixin(object):
+    # Like _MutableMappingMixin, but for sets.
+    def isdisjoint(self, other):
+        'Return True if two sets have a null intersection.'
```

Comment:
```suggestion
        """Return True if two sets have a null intersection."""
```

Reply:
Amusingly enough, that’s a copy from the Python standard library! They’re inconsistent too.